screens/authentication/signUp/SignUpScreen.jsx

```diff
@@ -19,16 +18,17 @@ import BouncyCheckbox from 'react-native-bouncy-checkbox';
 import {theme} from '../../../assets/theme/theme';
 import {GlobalDataContext} from '../../../contexts/context';
 import {setDataStorage} from '../../../utils/AsyncStorageApiUtils';
+import CustomTextInput from '../../../components/CustomInput/CustomTextInput';
 import styles from './style';
 
-const SignUpScreen = ({navigation}) => {
+const SignUpScreen = ({ navigation }) => {
   const [isChecked, setIsChecked] = useState(true);
   const [isPrivacyPolicy, setIsPrivacyPolicy] = useState(true);
   const [hidePassword, setHidePassword] = useState(true);
   const checkBoxText =
     'I would like to receive your newsletter and other promotional information.';
   const privacyPolicyText = 'Sipining up you accept the Privacy Policy.';
-  const {setUserData, setLoggedIn} = useContext(GlobalDataContext);
+  const { setUserData, setLoggedIn } = useContext(GlobalDataContext);
 
   useEffect(() => {
     values.isChecked = isChecked;
```

Comment:
this is the wrong code there is a setFieldValues function you can get it from useFormik and instead of declaring setState functions use setFieldValues and change values.isPrivacyPolicy and values.isChecked  

like a setFieldValues("isPrivacyPolicy", the necessary value)

add initialValues outside from useFormik and put only the variable in useFormik the same for function


instead of this sentence write validationSchema which will check all data
  if (data.email && data.password && data.name && data.isPrivacyPolicy)